src/content/docs/vulnerability-management/change-vulnerability-status.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: Change vulnerability status
+metaDescription: Use Vulnerability Management to overcome blindspots and assign remediation to developers as a security team.
+freshnessValidatedDate: never
+---
+
+import vmIgnoreEntityVulnList from 'images/security_screenshot-ignore-entity-vulnlist.webp'
+
+import vmIgnoreEntityDetails from 'images/security_screenshot-ignore-entity-details.webp'
+
+import vmIgnoreEntityForm from 'images/security_screenshot-ignore-entity-form.webp'
+
+import vmIgnoreBulkVulnList from 'images/security_screenshot-ignore-bulk-vulnlist.webp'
+
+import vmIgnoreBulkDetails from 'images/security_screenshot-ignore-bulk-details.webp'
+
+import vmIgnoreBulkForm from 'images/security_screenshot-ignore-bulk-form.webp'
+
+import vmAffectedEntityVulnList from 'images/security_screenshot-affected-entity-vulnlist.webp'
+
+import vmAffectedEntityDetails from 'images/security_screenshot-affected-entity-details.webp'
+
+import vmAffectedEntityFiltered from 'images/security_screenshot-affected-entity-filtered.webp'
+
+import vmAffectedEntityForm from 'images/security_screenshot-affected-entity-form.webp'
+
+import vmAffectedBulkVulnList from 'images/security_screenshot-affected-bulk-vulnlist.webp'
+
+import vmAffectedBulkSelection from 'images/security_screenshot-affected-bulk-selection.webp'
+
+import vmAffectedBulkDetails from 'images/security_screenshot-affected-bulk-details.webp'
+
+import vmAffectedBulkForm from 'images/security_screenshot-affected-bulk-form.webp'
+
+New Relic Vulnerability Management may detect an overwhelming amount of vulnerabilities across your system. Most of these vulnerabilities may need remediation, but many might be low risk, or have limited external exposure. You can manage your vulnerabilities by:
+
+* Assigning individual entities the Ignored status so their vulnerabilities no longer appear in Vulnerability Management
+* Ignoring a certain kind of vulnerability in bulk so it no longer appears across multiple entities 
+* Removing the Ignored status at a later date if you want to detect those vulnerabilities again
+
+## Ignore a vulnerability [#ignore-vulnerability]
+
+You can manually ignore vulnerabilities on a per entity basis. Once you set the status of an instance to Ignored, we’ll exclude this instance from the summary tiles and vulnerability list by default. Other instances of this vulnerability on other entities will still keep the Affected status.
+
+<CollapserGroup>
+    <Collapser
+        id="ignore-entity"
+        title="Ignore a vulnerability for a specific entity"
+    >
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All entities** > (select your entity) > **Vulnerability Management** > Click the **Vulnerabilities** tab, then select the vulnerability you want to ignore. If you don't see the vulnerability you want to manage, it might already have the Ignored status. 
```

Comment:
```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All entities** > (select your entity) > **Vulnerability Management** > Click the **Vulnerabilities** tab, then select the vulnerability you want to ignore. If you don't see the vulnerability you want to manage, it might already have the `Ignored` status. 
```